src/Websites/Platform/src/Bit.Websites.Platform.Client/Pages/Home/Products.razor.scss

```diff
@@ -117,7 +117,7 @@
 .template-terminal {
     display: flex;
     padding: 2rem;
-    color: #f0f0f0;
+    color: $bit-color-state-info-bg;
```

Comment:
it's not a proper color for this section, and generally, this is not a good approach.
the initial goal of this terminal was to simulate the default theme of the PowerShell so it is better to change the background color to the static value of `#060e2d` instead.
btw, remember to sort the final result based on the line length like all other CSS attributes.